variables.tf

```diff
@@ -70,3 +70,8 @@ variable "ec2_ingress_cidr_blocks" {
   type        = "string"
   default     = "0.0.0.0/0"
 }
+variable "ec2_ingress_sg_id" {
+  description = "Security groups to allow access to application ports."
```

Comment:
I'd suggest either rewording the description here to clarify that this only grants inbound access to port 80 or updating the well-known container port ingress rule to also grant access from resources assigned to specific security groups.